test/capi/capi_utils.hpp

```diff
@@ -152,6 +157,54 @@ static tc_sframe* make_sframe_double(const std::vector<std::pair<std::string, st
     return sf;
   }
 
-
+static bool check_equality_gl_sframe(
+  const turi::gl_sframe sf_gl, const turi::gl_sframe ref_gl) {
+
+  size_t num_columns_sf = sf_gl.num_columns();
+  size_t num_columns_ref = ref_gl.num_columns();
+
+  if (num_columns_sf == num_columns_ref) {
```

Comment:
Does this function check column names anywhere? Column types? (It's theoretically possible for SArrays of two different types to be elementwise-equal).

Reply:
I see types are checked below in the for loop. It doesn't look like the names are ever compared.

Reply:
Okay, I'll compare the column names too